### PR TITLE
finished making tabs

### PR DESCRIPTION
**Pull request by blueskye04**

### CODE REVIEW

**Comment by NathanDuPont** [comment]
This is awesome!!! I'll take a look at this one later tomorrow/on Saturday (I am VERY excited to see this in person). It looks like there are some merge conflicts with the other PR that just merged, could you go through and fix those? @blueskye04

**Review by blueskye04 — COMMENTED** [review]
done

**Review by blueskye04 — COMMENTED** [review]
done

File: Backend/app.py [hunk 1]
```diff
@@ -15,4 +15,16 @@ def parts():
...
\ No newline at end of file
+    print(request.form)
+    return "Hello!"
+
+
+@app.route('/uploader', methods = ['GET', 'POST'])
+def upload_file():
+   if request.method == 'POST':
+      f = request.files['my_file']
+      f.save(secure_filename(f.filename))
```
Comment (NathanDuPont) [inline Backend/app.py:26]: We will probably need to add the following line to import this function: 
```
from werkzeug.utils import secure_filename
```

Comment (NathanDuPont) [reply Backend/app.py:26]: This can be added between lines 1 and 2 (right after the existing import, but before `app = Flask(__name__)`

**Review by NathanDuPont — APPROVED** [review]

**Pull request merged**